src/main/java/tc/oc/occ/idly/IdlyManager.java

```diff
@@ -4,137 +4,123 @@
 import static net.kyori.adventure.sound.Sound.sound;
 import static net.kyori.adventure.text.Component.text;
 
-import java.time.Duration;
-import java.time.Instant;
+import com.google.common.base.Objects;
 import javax.annotation.Nullable;
 import net.kyori.adventure.audience.Audience;
 import net.kyori.adventure.sound.Sound;
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.format.NamedTextColor;
 import org.bukkit.entity.Player;
-import tc.oc.pgm.api.PGM;
 import tc.oc.pgm.api.match.Match;
 import tc.oc.pgm.api.player.MatchPlayer;
 import tc.oc.pgm.util.bukkit.OnlinePlayerMapAdapter;
 
 public class IdlyManager {
 
+  private static final int TICK_FREQUENCY = 10;
+  private static final int TICKS_PER_SECOND = 20;
+
   private final Idly plugin;
   private final IdlyConfig config;
-  private final OnlinePlayerMapAdapter<Instant> playerMovementCache;
-  private final OnlinePlayerMapAdapter<Instant> playerAfkCache;
+  private final OnlinePlayerMapAdapter<Integer> playerInactivityTickCache;
```

Comment:
```suggestion
  private final OnlinePlayerMapAdapter<Integer> playerInactivityTicks;
```
Cache is a term that usually refers to a faster way of accessing something that could otherwise be obtained from a main memory, this is really not a cache, this is our raw data, its misleading to call it a cache